src/algebra/group/conj.lean

```diff
@@ -49,3 +50,31 @@ end
 
 protected lemma monoid_hom.map_is_conj (f : α →* β) {a b : α} : is_conj a b → is_conj (f a) (f b)
 | ⟨c, hc⟩ := ⟨f c, by rw [← f.map_mul, ← f.map_inv, ← f.map_mul, hc]⟩
+
+namespace group
+
+/--
+Inner automorphisms of a group.
+-/
+def inner_aut (c : α) : α →* α :=
```

Comment:
Should we define `def conj : α →* (α ≃* α)` instead?

Reply:
Unfortunately, this won't work well with `to_additive` because it can't ignore `mul_equiv`.

Reply:
I didn't know about `mul_add`.  It turns out it already has `mul_aut.conj`, so I'll revert the changes to this file. :+1: 

The conjugation quandle of an abelian group is trivial, so I think I'm not concerned about `to_additive`.